src/core/components/creditCard/CreditCard.module.css

```
@@ -0,0 +1,58 @@
+.creditCard {
+  display: flex;
+  justify-content: center;
+
+  padding-top: 77px;
```

Stylelint (17.9.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)